straxen/plugins/veto_events.py

```diff
@@ -45,11 +43,6 @@ def infer_dtype(self):
         self.n_channel = (self.channel_range[1] - self.channel_range[0]) + 1
         return veto_event_dtype(self.name_event_number, self.n_channel)
 
-    def setup(self):
-        self.to_pe = straxen.get_to_pe(self.run_id,
-                                       self.config['gain_model_nv'],
-                                       self.n_channel)
-
```

Comment:
Right, was this ever needed actually?

Reply:
Only in a local version of mine. I forgot to remove it.